server_advanced.js

```diff
@@ -16,7 +16,8 @@ generator.handleResponses(app, {
     return spec;
   },
   specOutputPath: './test_spec.json',
-  mongooseModels: mongoose.modelNames()
+  mongooseModels: mongoose.modelNames(),
+  tags: ['foo']
```

Comment:
Something that's closer to a possible use case would be better - something like 

```ts
  tags: ['users']
```

just like in the readme 😄

Reply:
Just updated the example with meaningful paths.